python/hail/methods/statgen.py

```diff
@@ -13,7 +13,7 @@
 
 @handle_py4j
 @typecheck(dataset=MatrixTable,
-           maf=nullable(expr_numeric),
+           maf=nullable(oneof(Float32Expression, Float64Expression)),
```

Comment:
Won't this fail on Float32? Isn't it expecting a double?

Reply:
oh, hmmm. You're right! I've fixed the IBD parser to take float32s rather than restrict it here, though, since I feel like it ought to be able to handle that.